babel/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -35,10 +35,15 @@ SqlNode DateFunctionCall() :
         s = span();
         qualifiedName = new SqlIdentifier(unquotedIdentifier(), getPos());
     }
-    args = FunctionParameterList(ExprContext.ACCEPT_SUB_QUERY) {
-        quantifier = (SqlLiteral) args.get(0);
-        args.remove(0);
-        return createCall(qualifiedName, s.end(this), funcType, quantifier, args);
+    [
+        args = FunctionParameterList(ExprContext.ACCEPT_SUB_QUERY) {
+            quantifier = (SqlLiteral) args.get(0);
+            args.remove(0);
+            return createCall(qualifiedName, s.end(this), funcType, quantifier, args);
```

Comment:
I tried converting this to use `return SqlStdOperatorTable.DATE.createCall` but I couldn't get it to work properly. For some reason a testcase broke when I removed the `args.remove(0)` line, even if the quantifier wasn't used anywhere (ie I instead returned `SqlStdOperatorTable.DATE.createCall(getPos(), args);`.

Reply:
However it seems to be being parsed a function regardless since I had to remove the backticks for a previous test.

Reply:
It may be possible to just keep the line `args.remove(0)` and return the new function (that works for the current tests) but I'm not sure if it will work for other cases. For the cases that I've looked at `quantifier` is null.